src/integrationTests/secureSignal.test.ts

```diff
@@ -220,11 +220,14 @@ describe("when use script with SDK", () => {
 
 describe("getUid2AdvertisingTokenWithRetry", () => {
   test("should resolve with the result of the promise if it is successful", async () => {
-    const mockPromise = jest.fn(() => Promise.resolve("hello"));
+    const mockPromise = jest.fn();
```

Comment:
Either this should be a new test, or update the name. With this change, there's no test covering the happy path - if it just works the first time.

Reply:
Thanks, new test case added